server/src/test/java/org/elasticsearch/action/bulk/BulkOperationTests.java

```diff
@@ -206,11 +207,15 @@ public void testFailedBulkShardRequest() {
     @SuppressWarnings("unchecked")
     public void testInference() {
 
-        Map<String, Set<String>> fieldsForModels = Map.of(
-            INFERENCE_SERVICE_1_ID,
-            Set.of(FIRST_INFERENCE_FIELD_SERVICE_1, SECOND_INFERENCE_FIELD_SERVICE_1),
-            INFERENCE_SERVICE_2_ID,
-            Set.of(INFERENCE_FIELD_SERVICE_2)
+        FieldInferenceMetadata fieldInferenceMetadata = new FieldInferenceMetadata(
+            Map.of(
+                FIRST_INFERENCE_FIELD_SERVICE_1,
+                new FieldInferenceMetadata.FieldInferenceOptions(INFERENCE_SERVICE_1_ID, Set.of()),
```

Comment:
If we want this test to reflect production, the source field set should never be empty. In this case, it should contain `FIRST_INFERENCE_FIELD_SERVICE_1`

Reply:
Correct - this will be fixed when we start calculating inference for copy_to and multifields, as the way of retrieving the inference texts will start using the source information for fields 👍